lib/dogma/rule_builder.ex

```diff
@@ -41,6 +41,8 @@ defmodule Dogma.RuleBuilder do
 
   defmacro defrule(name, opts, [do: module_ast]) when is_list(opts) do
     opts = [{:enabled, true} | opts]
+    opts = Keyword.put_new(opts, :elixir, ">= 1.0.0")
```

Comment:
`:elixir_version` and `:elixir` are both fine. the latter has the slight advantage that the corresponding key in `mix.exs` is called `:elixir`. Since Dogma uses Mix.Config I thought you might want to keep consistent naming here. :grin: